src/rpc/evo.cpp

```diff
@@ -1303,7 +1303,8 @@ static UniValue BuildDMNListEntry(CWallet* pwallet, const CDeterministicMN& dmn,
     }
 #endif
 
-    auto metaInfo = mmetaman.GetMetaInfo(dmn.proTxHash);
+    CHECK_NONFATAL(mmetaman != nullptr);
```

Comment:
why NON-FATAL here? if it fails -> doesn't it mean that program is not initialized correctly at all?

Reply:
See [`lint-assertions.sh`](https://github.com/dashpay/dash/blob/38e70430b943021e6d4d6863ee53d7cb63c4fd16/test/lint/lint-assertions.sh#L23-L32).

A similar policy is visible for `NodeContext` (see [`rpc/blockchain.cpp`](https://github.com/dashpay/dash/blob/38e70430b943021e6d4d6863ee53d7cb63c4fd16/src/rpc/blockchain.cpp#L70-L77)), the lack thereof would signal even bigger problems.

We can fold these managers into `LLMQContext`-style structs or into `NodeContext` itself for cleaner errors but in the future and outside the scope of this PR.

Reply:
In theory, when can it happen? If CLI/UI will call this rpc before app is fully initialized? Hmm. Ok then